edsnlp/pipelines/misc/sections/sections.py

```diff
@@ -140,4 +146,7 @@ def __call__(self, doc: Doc) -> Doc:
         doc.spans["sections"] = sections
         doc.spans["section_titles"] = titles
 
+        if len(doc.ents) > 0:
+            self.tag_ents(sections)
+
```

Comment:
Is the check necessary ? spaCy's `.ents` property requires it to build ents from the iob tags, so I'd suggest skipping it

```suggestion
        self.tag_ents(sections)
```